Servers/manage-logs.sh

```
@@ -0,0 +1,204 @@
+    LOG_BASE_DIR="$LOCAL_LOGS_DIR"
+else
+    echo "No logs directory found. Creating $LOCAL_LOGS_DIR"
+    mkdir -p "$LOCAL_LOGS_DIR"
+    LOG_BASE_DIR="$LOCAL_LOGS_DIR"
+fi
+
+# Function to list all tenants with logs
+list_tenants() {
+    echo "=== Available Tenants ==="
+    if [ -d "$LOG_BASE_DIR" ]; then
+        ls -la "$LOG_BASE_DIR" | grep "^d" | awk '{print $9}' | grep -v "^\.$\|^\.\.$"
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)